addons/payment_adyen/models/payment_provider.py

```diff
@@ -125,7 +125,7 @@ def _build_url(_base_url, _version, _endpoint):
             _logger.exception(
                 "invalid API request at %s with data %s: %s", url, payload, error.response.text
             )
-            raise ValidationError("Adyen: " + _("The communication with the API failed."))
+            raise ValidationError("Adyen: " + error.response.json().get('message', ''))
```

Comment:
Keep the generic error message to provide some context, but append the eventual error message.

Something like that:
```suggestion
            raise ValidationError("Adyen: " + _("The communication with the API failed. Details: %s", error.response.json().get('message', '')))
```